### PR TITLE
[css-text][css-fonts] Testing writing system dependent rules

### PR DESCRIPTION
**Pull request by frivoal**
Related to https://github.com/w3c/csswg-drafts/issues/2015

### CODE REVIEW

File: css/css-text/writing-system/writing-system-font-001.html [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+<html lang=en>
+<meta charset=utf-8>
+<title>CSS test: font selection and writing sytem</title>
+<link rel=author title="Ambrose Li" href="http://port.ambroseli.ca/">
+<link rel=author title="Florian Rivoal" href="https://florian.rivoal.net">
+<link rel=help href="https://drafts.csswg.org/css-text-3/#languages">
+<link rel=help href="https://drafts.csswg.org/css-text-3/#script-tagging">
+<link rel=help href="https://drafts.csswg.org/css-fonts-4/#language-specific-support">
+<link rel=match href="reference/writing-system-font-001-ref.html">
+<meta name=assert content="An explicit writing system takes precedence over the content language for font selection">
```
Comment (foolip) [inline css/css-text/writing-system/writing-system-font-001.html]: This comment doesn't seem to match the test, which asserts (effectively) that `lang=foo` has no effect at all in this case.

Comment (frivoal) [reply css/css-text/writing-system/writing-system-font-001.html]: The test does not assert that `lang=foo` has no effect, but that if you have `lang=foo-bar` where `foo` is a language and `bar` is a writing system, it's `bar` that wins, not `foo`.

So unless I'm missing something, the comment is correct.

Comment (foolip) [reply css/css-text/writing-system/writing-system-font-001.html]: Oh, so the `-latn` bit is the important part. I've just revealed my ignorance about the area, but if nobody else will review, I will :)

File: css/css-text/writing-system/reference/writing-system-line-break-002-ref.html [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<!doctype html>
+<html lang=en>
+<meta charset=utf-8>
+<title>CSS test reference</title>
+<link rel=author title="Florian Rivoal" href="https://florian.rivoal.net">
+<style>
+</style>
+
+<p>The test passes if the question mark in the phrase below is on alone on the second line.
```
Comment (foolip) [inline css/css-text/writing-system/reference/writing-system-line-break-002-ref.html]:
```suggestion
<p>The test passes if the question mark in the phrase below is alone on the second line.
```

**Review by foolip — APPROVED** [review]

**Comment by frivoal** [comment]
> I'll merge as soon as w3c/csswg-drafts#3202 is merged.

That's done, so merging this as well.

**Comment by foolip** [comment]
@frivoal I noticed via https://bugs.chromium.org/p/chromium/issues/detail?id=905629 that my fixup commit ended up in the commit history, which I didn't intend. When merging PRs, please try to use the "squash" option in cases like this. (Nothing really bad happens if you forget, which I've done.)

**Comment by frivoal** [comment]
@foolip Yeah, I try to use squash when the commits are not meaningful, and regular merge (or rebase) when they are, but occasionally I forget. Sorry

**Pull request merged**